whois-api/src/main/java/net/ripe/db/whois/api/rdap/RdapObjectMapper.java

```diff
@@ -138,6 +137,14 @@ public Object mapSearch(final String requestUrl, final List<RpslObject> objects,
             }
         }
 
+        if(objects.size() == maxResultSize) {
```

Comment:
Is it possible that objects.size() can be larger than maxResultSize ?

Reply:
it can be smaller than max result size, this condition is used to add a notice in case it is above the limit. if the result is smaller we donot need to add a notice

Reply:
Thanks!